homeassistant/components/knx/__init__.py

```diff
@@ -367,11 +367,15 @@ def connection_config_tunneling(self):
         local_ip = self.config[DOMAIN][CONF_KNX_TUNNELING].get(
             ConnectionSchema.CONF_KNX_LOCAL_IP
         )
+        route_back = self.config[DOMAIN][CONF_KNX_TUNNELING].get(
+            ConnectionSchema.CONF_KNX_ROUTE_BACK
+        )
```

Comment:
```suggestion
        route_back = self.config[DOMAIN][CONF_KNX_TUNNELING][ConnectionSchema.CONF_KNX_ROUTE_BACK]
```

Reply:
you should run `black` locally and push instead of reverting the commit. 
See checklist of your initial issue comment.

Reply:
you are right. i wasn't familiar with `black`